### PR TITLE
FEATURE - Add force release for channels

### PR DESCRIPTION
**Pull request by dannypaz**
## Description
This PR adds a `force` flag to the `sparkswap wallet release` command that allows the user to force close all channels that are open w/ the exchange.

The Broker may run into an issue where it has open channels with the relayer, however the relayer may be down or offline. We want the broker to be able to close those channels HOWEVER the only way possible is by forcing the closing of the channel through the engines.

if the call to `sparkswap wallet release` fails, we will prompt the user to the user the `--force` flag if they understand that force releasing could take an extended amount of time AND this may add additional fees to the closing tx of the channel.

## Related PRs
List related PRs if applicable

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Link to Trello

### CODE REVIEW

**Comment by dannypaz** [comment]
[Add force option to smart release](https://trello.com/c/znlWaEau/651-add-force-option-to-smart-release)

File: broker-cli/commands/wallet.js [hunk 1]
```diff
@@ -300,16 +300,37 @@ function formatBalance (balance, status) {
...
+
+    const answer = await askQuestion(question)
 
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
-    await client.walletService.releaseChannels({market})
+    try {
+      await client.walletService.releaseChannels({ market, force })
+    } catch (e) {
+      logger.error(`Failed to release payment channels for ${market}`.red)
```
Comment (martineehrlich) [inline broker-cli/commands/wallet.js]: Do we want to show what the error is before we advise them to force close?

Comment (treygriffith) [reply broker-cli/commands/wallet.js]: 👍

File: broker-cli/commands/wallet.js [hunk 2]
```diff
@@ -521,7 +546,8 @@ module.exports = (program) => {
     .command(`wallet ${SUPPORTED_COMMANDS.NETWORK_STATUS}`, 'Payment Channel Network status for trading in different markets')
     .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
     .command(`wallet ${SUPPORTED_COMMANDS.RELEASE}`, 'Closes channels open on the specified market')
-    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
+    .option('--market <marketName>', 'Relevant market name', validations.isMarketName, null, true)
+    .option('--force [force]', 'Relevant market name', null, false)
```
Comment (treygriffith) [inline broker-cli/commands/wallet.js]: I don't think force takes a parameter, so it should just be `--force`, not `--force [force]`, right? Unless I'm misunderstanding the caporal syntax

File: broker-cli/proto/broker.proto [hunk 3]
```diff
@@ -921,6 +921,8 @@ message CommitRequest {
 message ReleaseChannelsRequest {
   // Market to close channels on
   string market = 1;
+  // Force channels to close which may result in funds being locked for additional time and fees
```
Comment (treygriffith) [inline broker-cli/proto/broker.proto]: Add a note that it includes inactive and pending channels

File: broker-cli/commands/wallet.js (hunk 1 above)
Comment (dannypaz) [reply broker-cli/commands/wallet.js]: @treygriffith @martineehrlich the error gets rethrown after the text.

File: broker-cli/commands/wallet.js (hunk 2 above)
Comment (dannypaz) [reply broker-cli/commands/wallet.js]: nope! you are all good. thanks for the catch

File: broker-cli/commands/wallet.js (hunk 1 above)
Comment (martineehrlich) [reply broker-cli/commands/wallet.js]: Yes, but I was suggesting that we should show/log what the error is here, and still throw after.

File: broker-cli/commands/wallet.js [hunk 4]
```diff
@@ -300,16 +300,37 @@ function formatBalance (balance, status) {
...
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
-    await client.walletService.releaseChannels({market})
+    try {
+      await client.walletService.releaseChannels({ market, force })
+    } catch (e) {
+      logger.error(`Failed to release payment channels for ${market}`.red)
+
+      if (!force) {
+        logger.info('You can force the release of funds using the `--force` option, however')
```
Comment (treygriffith) [inline broker-cli/commands/wallet.js]: [nit] maybe use a multi-line string here

File: broker-cli/commands/wallet.js [hunk 5]
```diff
@@ -521,7 +546,8 @@ module.exports = (program) => {
     .command(`wallet ${SUPPORTED_COMMANDS.NETWORK_STATUS}`, 'Payment Channel Network status for trading in different markets')
     .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
     .command(`wallet ${SUPPORTED_COMMANDS.RELEASE}`, 'Closes channels open on the specified market')
-    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
+    .option('--market <marketName>', 'Relevant market name', validations.isMarketName, null, true)
+    .option('--force', 'Relevant market name', null, false)
```
Comment (treygriffith) [inline broker-cli/commands/wallet.js]: this description for `--force` looks wrong

Comment (dannypaz) [reply broker-cli/commands/wallet.js]: @treygriffith it is sooo wrong. thanks for the catch

File: broker-cli/commands/wallet.js (hunk 4 above)
Comment (dannypaz) [reply broker-cli/commands/wallet.js]: @treygriffith the formatting was easier w/ individual lines

**Pull request merged**